docs/building.md

```diff
@@ -0,0 +1,128 @@
+pip3 install --upgrade meson==0.64.1 markdown==3.4.1 markupsafe==2.1.1 jinja2==3.1.2 pygments==2.13.0 toml==0.10.2 typogrify==2.0.7 tomli==2.0.1
+```
+
+After installing the prerequisites listed above, download the code and init the submodules
+```
+git clone https://github.com/cypherstack/stack_wallet.git
+cd stack_wallet
+git submodule update --init --recursive
+
+```
+
+Run prebuild script
+
+```
+cd scripts
+./prebuild.sh
+// when finished go back to the root directory
+cd ..
+```
+
+Remove pre-installed system libraries for the following packages built by cryptography plugins in the crypto_plugins folder: `boost iconv libjson-dev libsecret openssl sodium unbound zmq`.  You can use
+```
+sudo apt list --installed | grep boost
+```
+for example to find which pre-installed packages you may need to remove with `sudo apt remove`.  Be careful, as some packages (especially boost) are linked to GNOME (GUI) packages: when in doubt, remove `-dev` packages first like with
+```
+sudo apt-get remove '^libboost.*-dev.*'
+```
+<!-- TODO: configure compiler to prefer built over system libraries. Should already use them? -->
+
+Building plugins for Android
+```
+cd scripts/android/
+./build_all.sh
+// when finished go back to the root directory
+cd ../..
+```
+
+Building plugins for Linux
+
+```
+cd scripts/linux/
+./build_all.sh
+// when finished go back to the root directory
+cd ../..
+```
+
+## Running
+### Android
+Plug in your android device or use the emulator available via Android Studio and then run the following commands:
+```
+flutter pub get
+flutter run android
+```
+
+Note on Emulators: Only x86_64 emulators are supported, x86 emulators will not work
+
+### Linux
+Plug in your android device or use the emulator available via Android Studio and then run the following commands:
+```
+flutter pub get Linux
+flutter run linux
+```
+
+## Android Studio
+Android Studio is the recommended IDE for development, not just for launching on Android devices and emulators but also for Linux desktop development. 
+
+Follow instructions here [https://developer.android.com/studio/install#linux](https://developer.android.com/studio/install#linux) or install via snap:
+```
+# setup android studio
+sudo apt install -y openjdk-11-jdk
+sudo snap install android-studio --classic
+```
+
+Use Tools > SDK Manager to install the SDK Tools > Android SDK (API 30), SDK Tools > NDK, SDK Tools > Android SDK command line tools, and SDK Tools > CMake
+
+Then install the Flutter plugin and restart the IDE.  In Android Studio's options for the Flutter language, enable auto format on save to match the project's code style.  If you have problems with the Dart SDK, make sure to run `flutter` in a terminal to download it (use `source ~/.bashrc` to update your environment variables if you're still using the same terminal from which you ran `setup.sh`)
+
+Make a Pixel 4 (API 30) x86_64 emulator with 2GB of storage space for emulation
+
+#### Fedora (37) (Work In Progress)
```

Comment:
Let's get these Fedora build instructions in here!  Love that you've shown how to build on Fedora in the previous PR